vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Installation tree layout
+description: This article describes the layout conventions used by vcpkg when copying build output into the installation directory.
+author: vicroms
+ms.author: viromer
+ms.date: 9/21/2023
+---
```

Comment:
```suggestion
ms.topic: reference
---
```
?

Reply:
Now that Augustin changed the default document type to `reference` we don't need to set it explicitly.